tools/githooks/windows/pre-push

```diff
@@ -1,10 +0,0 @@
-#!/bin/sh

```

Comment:
Shouldn't this file be replaced with one that actually does the pre-push check? Or does just removing this run the general one instead?

Reply:
The original pre-push in tools/githooks works on all platforms now that Windows has all the checks are working.

Granted people on Windows need to run `gclient sync` after this change to re-run tools/githooks/setup.py, which will modify the git config to use the common pre-push file on Windows. This will not be necessary on other platforms.